.azure/OneBranch.Official.yml

```diff
@@ -75,9 +75,6 @@ extends:
       - container: windows_build_container2
         image: 'cdpxb7b51c2f738e43e48f7605d9a8e5f6d700.azurecr.io/b7b51c2f-738e-43e4-8f76-05d9a8e5f6d7/official/msquicbuild:${{ parameters.WindowsContainerImage2DockerTag }}'
         type: Windows
-      - container: linux_build_container # Default container
```

Comment:
You must specify the default container, with the name `linux_build_container`. Other onebranch stuff depends on it.